pgjdbc/src/main/java/org/postgresql/core/CommandCompleteParser.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2018, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.core;
+
+import org.postgresql.util.GT;
+import org.postgresql.util.PSQLException;
+import org.postgresql.util.PSQLState;
+
+/**
+ * Parses {@code oid} and {@code rows} from a {@code CommandComplete (B)} message (end of Execute).
+ */
+public final class CommandCompleteParser {
```

Comment:
The diff looks odd

Reply:
I agree. I tried to rebase (as you had suggested earlier) and all of a sudden your commit of the CommandComplete stuff showed up.
I am accustomed to doing merges rather than rebases, so it is quite likely I have simply done something wrong. However, I have no idea how to undo it now.

Reply:
This is what helps me in similar cases: https://gist.github.com/iansheridan/870778/28f1a5a2b35a14eea5cac671d126b092e8448d9c#info

>git reflog Use this to recover from major fuck ups!

Reply:
@vlsi, I think this might be a github flub. It looks correct from my fork:
https://github.com/pgjdbc/pgjdbc/compare/master...bokken:int_collection